API/Tensorflow/native_libs/src/ops/Binder.h

```diff
@@ -36,6 +36,8 @@ class Binder : public Hashable {
     /*
      * This method should add the Node to the provided graph
      * AND add all of the Node's Outputs that haven't been freed yet as well.
+     * It returns the operation computing this node (if any, otherwise nullptr),
```

Comment:
This method doesn't seem to return anything

Reply:
Right, great catch - I added this comment during the refactor that I later reverted because some assumptions changed. Certainly should delete this, thanks.